provision/acc_provision/apic_provision.py

```diff
@@ -469,6 +485,21 @@ def nested_dom(self):
                 ]
             }
         }
+        if self.is_kube():
+            kubeapi_vlan = self.config["net_config"]["kubeapi_vlan"]
+            data["vmmUsrCustomAggr"]["children"].append(
+                {
+                    "fvnsEncapBlk": {
+                        "attributes": {
+                            "from": "vlan-%d" % kubeapi_vlan,
+                            "to": "vlan-%d" % kubeapi_vlan,
+                        }
+                    }
+                })
+        if self.is_cf():
+            # TODO This should be required for Kubernetes as well
```

Comment:
Without this option, the ESX vSwitch drops the reply packets on the service VLAN because the source MAC (node service MAC) is different from that of the VM's interface.